Paintroid/src/main/res/values/arrays.xml

```diff
@@ -56,4 +56,12 @@
         <item>@string/text_tool_dialog_font_dubai</item>
         <item>@string/text_tool_dialog_font_arabic_stc</item>
     </string-array>
+
+    <integer-array name="text_tool_size_array" translatable="false">
```

Comment:
should be `pocketpaint_text_tool_size_array`, otherwise we might collisions in Catroid.